### PR TITLE
Create encode.md for Python Strings .encode()

### LINKED ISSUE #3272: [Term Entry] Python Strings .encode()
### Reason/inspiration (optional)

We would like a new term entry on the `.encode()` method for Python: [Strings](https://www.codecademy.com/resources/docs/python/strings). The entry should go in a new file under `docs/content/python/concepts/strings/terms/encode/encode.md`.

The entry should describe what the method does. Be sure to include the following sections:

- A "Syntax" section with a ` ```pseudo` code snippet.
- An "Example" section with a ` ```py `code snippet.
- A “Codebyte Example” section with a ` ```codebyte/python ` snippet.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by reganmarie**
### Description

- Closes #3272 
- Created encode.md on docs/content/python/concepts/strings/terms/encode/encode.md
- Added Title, Description, Subjects, Tags, and Catalog Content
- Added description, syntax, example, and codebyte example
- Ran through 3 online plagiarism checkers to ensure 100% unique wording and description, passed with 100%
- Followed content and style guidelines -Checked for spelling and grammar errors
- Read over initial draft to correct for clarity and readability

### Type of Change

- Adding a new entry

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [X] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/python/concepts/strings/terms/encode/encode.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+- `errors` (optional): How to handle errors if the string cannot be encoded. There are six error responses:
+  - `strict`: A `UnicodeDecodeError` will be raised on failure. If no error handling is specified, this is the default.
+  - `ignore`: Ignore characters that cannot be encoded.
+  - `replace`: Replace the characters that cannot be encoded with a question mark. -`xmlcharrefreplace`: If there is a character that cannot be encoded, an XML character reference will be inserted in its place.
+  - `backslashreplace`: If there is a character that cannot be encoded, a \uNNNN will be inserted in its place.
+  - `namereplace`: If there is a character that cannot be encoded, a \N{...} will be inserted in its place.
+
+## Example
+
+In the following example, the `.encode()` method is called twice. The first call does not specify the optional parameters, so the string is encoded in UTF-8 and will throw a `UnicodeDecodeError` if there is failure. The second call specifies that the string will be encoded using `ASCII` and that errors will be handled using `xmlcharrefreplace`.
```
Comment (caupolicandiaz) [inline content/python/concepts/strings/terms/encode/encode.md:34]:
```suggestion
In the following example, the `.encode()` method is called twice. The first call does not specify the optional parameters, so the string is encoded in UTF-8 and will throw a `UnicodeDecodeError` if there is failure. The second call specifies that the string will be encoded using `ascii` and that errors will be handled using `xmlcharrefreplace`.
```

File: content/python/concepts/strings/terms/encode/encode.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+```codebyte/python
+example_a = 'Tèlepöne?'
+example_b = 'Pöè!'
+
+default_example = example_a.encode()
+utf_16_example = example_b.encode(encoding='utf-16', errors='ignore')
+
+print("Default example:", default_example)
+print("ASCII example:", ascii_example)
```
Comment (caupolicandiaz) [inline content/python/concepts/strings/terms/encode/encode.md:66]:
```suggestion
print("UTF-16 example:", utf_16_example)
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@reganmarie I've completed my initial review. Please update the entry, and tag me when you've finished making the changes. Thanks

**Comment by reganmarie** [comment]
@caupolicandiaz Made the following requested changes! 
- Changed line 29 to the `\uNNNN` characters
- Changed line 30 to the `\N{...}` characters
- Changed line 34 to ascii rather than ASCII
- Changed line 66 to ("UTF-16 example:", utf_16_example)

**Review by caupolicandiaz — APPROVED** [review]
@reganmarie This is ready for a second review.

**Comment by reganmarie** [comment]
Thank you @caupolicandiaz! 😊

**Review by SSwiniarski — APPROVED** [review]
@reganmarie Fixed one typo and it looks good to merge.

**Comment by reganmarie** [comment]
Thank you @SSwiniarski! 😊

**Pull request merged**